pkgs/node/src/chain.zig

```diff
@@ -149,6 +157,14 @@ pub const BeamChain = struct {
         return vote;
     }
 
+    // TODO: right now validator indepdently publishes to the network but move the gossip
```

Comment:
@noopur23 addressing this also in the same issue as the publishblock todo comment one